### ISSUE TITLE
Yolo 8 or 11 pose estimation model modifications for far object from camera

### ISSUE THREAD

**Issue by hvmatrix** [ultralytics/ultralytics#17757]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hi,
I am working on pose-estimation model to detect the person and its pose. For ceiling mounted and wall mounted camera while I'm using the model like yolo8-pose or yolo11-pose, it can not detect the person which is far from camera. But while using detection model it is able to detect the far person also. Why this is happening? I have tried to change confidence threshold of keypoints also. but it's not working. What can i do to modify the code and where to modify the code so I can get better results.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @hvmatrix, thank you for reaching out to Ultralytics 🚀! We appreciate your interest in enhancing our pose estimation models for your specific use case.

For immediate support with any questions or configurations, we suggest checking the [Docs](https://docs.ultralytics.com) where detailed [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples are available. They might provide insights into addressing technical difficulties.

As your issue appears to relate to model performance 🐛, could you please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/)? This would greatly help us in diagnosing and resolving it efficiently. Additionally, sharing dataset image examples and any training logs could be beneficial if this is about custom training adjustments.

For more interactive assistance, join our vibrant community 😃:
- Engage in live discussions on [Discord](https://ultralytics.com/discord) 🎧.
- Explore in-depth topics on [Discourse](https://community.ultralytics.com).
- Dive into interesting threads and share your experiences on our [Subreddit](https://reddit.com/r/ultralytics).

## Upgrade

Ensure you are using the latest version of the `ultralytics` package to avoid outdated issues. Update with the command below in a [**Python>=3.8**](https://www.python.org/) environment supported by [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

Rest assured, this automated response is just the beginning. An Ultralytics engineer will review your query soon to provide further assistance. Thank you for your patience and understanding!

**Comment by hvmatrix** [comment]
@Y-T-G, Ok so it will not work if i will change internal structure? You wanna say that?

**Comment by Y-T-G** [comment]
No. It's a problem with generalization. The model doesn't perform well on angles and views it wasn't trained on.

**Comment by hvmatrix** [comment]
Ok got it. Thank you for clearing my doubts.
